contracts/token/ERC721/ERC721Full.sol

```diff
@@ -11,8 +10,14 @@ import "./ERC721Metadata.sol";
  *
  * See https://eips.ethereum.org/EIPS/eip-721
  */
-contract ERC721Full is ERC721, ERC721Enumerable, ERC721Metadata {
-    constructor (string memory name, string memory symbol) public ERC721Metadata(name, symbol) {
-        // solhint-disable-previous-line no-empty-blocks
+contract ERC721Full is ERC721Enumerable, ERC721Metadata {
```

Comment:
Are we reviewing this name? Is there a list of pending issues for 3.0?

Reply:
Thanks for the suggestion! I created https://github.com/OpenZeppelin/openzeppelin-contracts/issues/2086